cmd/storage-datatypes.go

```diff
@@ -153,6 +153,9 @@ type FileInfo struct {
 	VersionPurgeStatus            VersionPurgeStatusType
 
 	Data []byte // optionally carries object data
+
+	parallelVersions int
+	successorModTime time.Time
```

Comment:
These are not carried across when FileInfo is serialized, just so you are aware.

Reply:
@vadmeste you don't need these datatypes over the network?

Reply:
I don't *think* they need to be transmitted, but I could be wrong.

Reply:
NumVersions & SuccessorModTime are variable and they should not be stored in the disk, but it looks like FileInfo is transmitted over network, like for storage-interface.ReadVersion() -> FileInfo

Reply:
Okay, I checked. FileInfo is not stored in the disk, it is only in memory and between nodes, made the necessary changes